ui/StatusQ/src/StatusQ/Popups/Dialog/StatusDialog.qml

```diff
@@ -1,5 +1,5 @@
 import QtQuick 2.15
-import QtQuick.Controls 2.15
+import QtQuick.Controls.Universal 2.15
```

Comment:
Aren't we losing the native animations/sheets everywhere on mobile with this import?